apps/i18n/common/en_us.json

```diff
@@ -976,6 +978,8 @@
   "histogram": "Histogram",
   "homepageHeading": "My Dashboard",
   "howItWorks": "How it Works (View Code)",
+  "iAgree": "I agree",
+  "iAmA": "I am a",
```

Comment:
partial sentences will not work well for translations. I would suggest: 
```
"I am a ..."
```
only slightly different, but prompts translators to move the ... in languages where the word order might be something like "a ... am I" (my new trick for this will be to imagine Yoda saying everything)

Reply:
Oh, good call. I'll do that for now, and will double check with Tess if we want to keep this label as "I am a ..." because you're right that translations will be tricky